test/karma.conf.js

```diff
@@ -16,7 +16,7 @@ module.exports = function(config) {
       clearContext: false,
       qunit: {
         showUI: true,
-        testTimeout: 30000
+        testTimeout: 5000
```

Comment:
playback tests are coded to 50s in beforeEach, so this is just a default for other tests